include/hc_printf.hpp

```diff
@@ -241,14 +276,14 @@ static inline void processPrintfBuffer(PrintfPacket* gpuBuffer) {
 
   if (gpuBuffer == NULL) return;
   unsigned int bufferSize = gpuBuffer[0].data.ui;
-  unsigned int cursor = gpuBuffer[1].data.ui;
-  unsigned int numPackets = ((bufferSize<cursor)?bufferSize:cursor) - 5;
+  unsigned int cursor = gpuBuffer[3].data.ai[0];
+  unsigned int numPackets = ((bufferSize<cursor)?bufferSize:cursor) - 4;
```

Comment:
@aaronenyeshi  i'm a bit afraid of all these magic numbers. could you try change them into enums?